units.libs.lua

```diff
@@ -183,7 +183,7 @@ StaticLibrary {
 		Filters = {
 			{ Pattern = "[/\\]windows[/\\]"; Config = { "win32-*", "win64-*" } },
 			{ Pattern = "[/\\]macosx[/\\]"; Config = "mac*-*" },
-			{ Pattern = "[/\\]x11[/\\]"; Config = { "x11-*" } },
+			{ Pattern = "[/\\]X11[/\\]"; Config = { "x11-*" } },
```

Comment:
So I tested this on a Linux (Debian jessie) machine today and for me the directory is lower case and everything builds just fine